service/storage_service.cc

```diff
@@ -1944,9 +1946,11 @@ future<> storage_service::bootstrap(std::unordered_set<token>& bootstrap_tokens,
         set_mode(mode::BOOTSTRAP);
         slogger.debug("bootstrap: rbno={} replacing={}", bootstrap_rbno, is_replacing());
 
-        // Wait until we know tokens of existing node before announcing replacing status.
-        slogger.info("Wait until local node knows tokens of peer nodes");
-        _gossiper.wait_for_range_setup().get();
+        if (!raft_topology_change_enabled()) {
```

Comment:
Is `storage_service::bootstrap()` even called on raft-topology path?

Reply:
Okay, now I see
```cpp
co_await _sys_ks.local().set_bootstrap_state(db::system_keyspace::bootstrap_state::COMPLETED);
```
in `storage_service::join_token_ring` before 
```cpp
if (should_bootstrap()) {
  [...]
  co_await bootstrap(bootstrap_tokens, cdc_gen_id, ri);
}
```
So no, it's not called on raft-topology path.